### ISSUE TITLE
Private properties are seen as public when writing own rule (PHP7.4)

### ISSUE THREAD

**Issue by Erkens** [phpmd/phpmd#735]
- PHPMD version: 2.8.1
- PHP Version: 7.4.2
- Installation type: composer
- Operating System / Distribution & Version: AlpineLinux (official PHP docker image)

### Current Behavior
I have created a simple rule that checks if there are no public properties, however, with PHP7.4 all (typed) properties are now seen as public:
`class TestClass { private string $id; }`

And (a simplified version of) my rule:
```php
final class PublicPropertiesRule extends AbstractRule implements ClassAware
{
    public function apply(AbstractNode $classNode)
    {
        $realClassNode = $classNode->getNode();

        $properties = $realClassNode->getProperties();
        foreach ($properties as $property) {
            if (!$property->isPublic()) {
                continue;
            }
            // we should not reach here with the sample TestClass class....
        }
    }
}
```

### Expected Behavior
Property should be marked as private and not public

**Comment by kylekatarnls** [comment]
Thank you for the report, it will be handled soon in PDepend:
https://github.com/pdepend/pdepend/issues/474
